### ISSUE TITLE
read kubeconfig from env KUBECONFIG

### ISSUE THREAD

**Issue by diemus** [lensapp/lens#483]
I have multiple kubeconfig in my computer, so I move them into a dir to manage them. not the default `~/.kube/config`
```
~/.kube/cluster_config/xxx_cluster.yaml
```
and set env for them to combine
```
export KUBECONFIG=$KUBECONFIG:$HOME/.kube/cluster_config/xxx_1.yaml
export KUBECONFIG=$KUBECONFIG:$HOME/.kube/cluster_config/xxx_2.yaml
export KUBECONFIG=$KUBECONFIG:$HOME/.kube/cluster_config/xxx_3.yaml
```

I can not found those kubeconfig in lens, the select option is empty. and there is not way to set env path for lens. some of my cluster's ip often change, i don't want manually added cluster because it seems can not change API Address after you add a cluster

**Comment by wfhartford** [comment]
I have a similar issue I manage several different clusters using different kubeconfig files. I would love to be able to specify a custom kubeconfig file when adding a cluster. In my case, the custom option would be inconvenient because the authentication credentials in the kubeconfig files expire daily and must be replaced.

**Comment by jnummelin** [comment]
We're working on to fix this at #466. It's quite a big and deep refactoring but hopefully we'll be able to push this to 3.6 series.

**Comment by nevalla** [comment]
With `3.6.0` you can select kubeconfig files from the local filesystem. You can test this feature already with 3.6.0 release candidate.
